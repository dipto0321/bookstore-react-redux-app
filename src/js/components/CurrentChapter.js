import React from 'react';
import PropTypes from 'prop-types';

const CurrentChapter = ({ currentChapter }) => (
  <div>
    <p className="current-chapter">CURRENT CHAPTER</p>
    <p className="current-lesson">
      { currentChapter === '' ? 'Not started' : `Chapter ${currentChapter}` }
    </p>
  </div>
);

CurrentChapter.propTypes = {
  currentChapter: PropTypes.string.isRequired,
};

export default CurrentChapter;
