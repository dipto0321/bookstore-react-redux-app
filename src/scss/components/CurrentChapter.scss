.current-chapter {
  opacity: 0.5;
  font-family: "Roboto Slab", serif;
  font-size: 13px;
  font-weight: 300;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  color: #121212;
}

.current-lesson {
  font-family: "Roboto Slab", serif;
  font-size: 20px;
  font-weight: 300;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: -0.4px;
  color: #121212;
}
