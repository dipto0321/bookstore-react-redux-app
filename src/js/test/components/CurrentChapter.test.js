import React from 'react';
import { shallow } from 'enzyme';
import CurrentChapter from '../../components/CurrentChapter';

test('CurrentChapter should render correctly', () => {
  const wrapper = shallow(<CurrentChapter currentPage="" />);
  expect(wrapper).toMatchSnapshot();
});
