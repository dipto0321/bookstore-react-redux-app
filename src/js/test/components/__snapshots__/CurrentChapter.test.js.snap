// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentChapter should render correctly 1`] = `
<div>
  <p
    className="current-chapter"
  >
    CURRENT CHAPTER
  </p>
  <p
    className="current-lesson"
  >
    Chapter 17
  </p>
</div>
`;
